polaris/tasks/ocean/external_gravity_wave/forward.yaml

```diff
@@ -0,0 +1,69 @@
+ocean:
+  time_management:
+    config_stop_time: none
+    config_run_duration: {{ run_duration }}
+  time_integration:
+    config_dt: {{ dt }}
+    config_time_integrator: {{ time_integrator }}
+
+mpas-ocean:
+  run_modes:
+    config_ocean_run_mode: forward
+  decomposition:
+    config_block_decomp_file_prefix: graph.info.part.
+  debug:
+    config_disable_thick_vadv: true
+    config_disable_thick_sflux: true
+    config_disable_vel_coriolis: true
+    config_disable_vel_hmix: true
+    config_disable_vel_surface_stress: true
+    config_disable_vel_explicit_bottom_drag: true
+    config_disable_vel_vmix: true
+    config_disable_vel_vadv: true
+    config_disable_tr_hmix: true
+    config_disable_tr_vmix: true
+    config_disable_tr_sflux: true
+    config_disable_tr_nonlocalflux: true
```

Comment:
```suggestion
```

`config_disable_tr_all_tend` means that all of these tendencies will be disabled.